packages/solo/src/web.js

```diff
@@ -194,6 +200,66 @@ export async function makeHTTPListener(
       .catch(_ => {});
   });
 
+  app.get('/connections', async (req, res) => {
+    if (!(await validateAccessToken(req))) {
+      res.json({ ok: false, rej: 'Unauthorized' });
+      return;
+    }
+
+    res.json({
+      ok: true,
+      connections,
+    });
+  });
+
+  app.post('/publish-bundle', async (req, res) => {
```

Comment:
This is not tested. It would be tested by `solo/test/test-home.js` if that worked. There’s probably something wrong here with request or response size limits, despite the limit set above. However, this is just for the `http` connection type and isn’t employed when submitting to a Cosmos chain, fake or otherwise.